### PR TITLE
Add option ``fitted_kws`` in ``distplot``

### PR DESCRIPTION
**Pull request by gsagnol**
This option alo=lows to pass some kw_args arguments to the 
`fit( )` function, such as `floc=0`.

### CODE REVIEW

File: seaborn/distributions.py [hunk 1]
```diff
@@ -236,7 +240,7 @@ def distplot(a, bins=None, hist=True, kde=True, rug=False, fit=None,
         clip = fit_kws.pop("clip", (-np.inf, np.inf))
         bw = stats.gaussian_kde(a).scotts_factor() * a.std(ddof=1)
         x = _kde_support(a, bw, gridsize, cut, clip)
-        params = fit.fit(a)
+        params = fit.fit(a,**fitter_kws)
```
Comment (mmagnuski) [inline seaborn/distributions.py]: space after comma

**Comment by mmagnuski** [comment]
I've had a very similar situation when I needed to use `floc=0` because otherwise gamma distribution would not fit well. I would like such option (passing args to `fit`) but having `fit_kws` and `fitter_kws` would be confusing...

**Comment by gsagnol** [comment]
Yes. The names might not be optimal. If you have a better idea, feel free to modify ! I think it might be a useful option, though.

**Comment by mmagnuski** [comment]
Actually, I think `fit_kws` should be the ones passed to `fit()` and something like `plotfit_kws` should be those passed to `plot()` the fitted distribution. But that would require a deprecation etc. - not sure it is worth it. I normally fit the distribution myself and then plot it to the distplot axis.

**Comment by mwaskom** [comment]
Another option is to make your own object and pass that in, e.g..

```python
class normfit(stats.norm):
    def fit(*args, **kwargs)
         return super(normfit, self).fit(loc=0, *args, **kwargs)
sns.distplot(x, fit=normfit)
```

Not super convenient for one-off use, but completely flexible.

**Comment by mwaskom** [comment]
>  I normally fit the distribution myself and then plot it to the distplot axis.

I think this is the right approach. `distplot` is meant to be a function for quickly checking aspects of the distribution, not a general-purpose frontend for fitting parametric distributions.

**Pull request closed without merging**